simplemonitor/Alerters/ntfy.py

```diff
@@ -78,6 +93,12 @@ def send_alert(self, name: str, monitor: Monitor) -> None:
         subject = self.build_message(AlertLength.NOTIFICATION, alert_type, monitor)
         body = self.build_message(AlertLength.FULL, alert_type, monitor)
 
+        # prefix icon to subject when relevant
+        if self.ntfy_icon_prefix and subject.endswith("failed"):
+            subject = f"{chr(int(self.ntfy_icon_failed, 16))} {subject}"
```

Comment:
Maybe do the `chr(int(...))` dance in `__init__` so that if it's an invalid char it fails during loading of the config rather than when trying to send an alert :)